python/ray/serve/_private/http_proxy.py

```diff
@@ -36,6 +36,9 @@
 SOCKET_REUSE_PORT_ENABLED = (
     os.environ.get("SERVE_SOCKET_REUSE_PORT_ENABLED", "1") == "1"
 )
+REQUEST_PROCESSING_TIMEOUT_S = (
+    float(os.environ.get("REQUEST_PROCESSING_TIMEOUT_S", 0)) or None
```

Comment:
```suggestion
    float(os.environ.get("SERVE_REQUEST_PROCESSING_TIMEOUT_S", 0)) or None
```

Reply:
I changed `REQUEST_PROCESSING_TIMEOUT_S` to `SERVE_REQUEST_PROCESSING_TIMEOUT_S`.